s3_upload/utils/log.py

```
@@ -35,31 +39,82 @@
     logger.addHandler(file_handler)
 
+    logger.info(
+        "Initialised log FileHandler, setting log output to %s", log_file
+    )
+
+    clear_old_logs(logger=logger, log_dir=log_dir, backup_count=5)
+
     return logger
 
 
+def clear_old_logs(logger, log_dir, backup_count) -> None:
```

Ruff (0.8.0)

Missing type annotation for function argument `logger`

(ANN001)